modulefiles/build_hera_gnu.lua

```diff
@@ -7,10 +7,10 @@ whatis([===[Loads libraries needed for building the UFS SRW App on Hera using GN
 
 prepend_path("MODULEPATH", "/scratch2/NCEPDEV/stmp1/role.epic/installs/gnu/modulefiles")
 prepend_path("MODULEPATH", "/scratch2/NCEPDEV/stmp1/role.epic/installs/openmpi/modulefiles")
-prepend_path("MODULEPATH", "/scratch2/NCEPDEV/stmp1/role.epic/spack-stack/spack-stack-1.6.0_gnu13/envs/fms-2024.01/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/test.remove/install/modulefiles/Core")
 
-load("stack-gcc/13.3.0")
-load("stack-openmpi/4.1.6")
+load("stack-gcc/9.2.0")
```

Comment:
Is there a reason the gnu stack was downgraded to use gcc/9.2.0 and stack-openmpi/4.1.5?..

Reply:
@mkavulich i assume this change was just for your testing? i believe it should indeed point to `/scratch2/NCEPDEV/stmp1/role.epic/spack-stack/spack-stack-1.6.0_gnu13/envs/fms-2024.01/install/modulefiles/Core` for the gnu stack that contains the new met/metplus versions. is this correct, @RatkoVasic-NOAA?